### ISSUE TITLE
Export C# Nunit Compiler Error for Pause Commands

### ISSUE THREAD

**Issue by denniskreutz** [SeleniumHQ/selenium-ide#1025]
## 🐛 Bug Report
Hello guys,

when you add a pause element/command and try to export the steps to a c# nunit cs File. This File has compile errors because the Exception  object is not there.

Exported code: 
```c#
  try {
      Thread.Sleep(3000);
    } catch {
      Console.WriteLine("{0} Exception caught.", e);
    }
```
fixed code by myself:
```c#
   try
        {
            Thread.Sleep(3000);
        }
        catch **(Exception e)**
        {
            Console.WriteLine("{0} Exception caught.", e);
        }
```
as you can see the (Exception e) is missing in your exported c# code.

Could you pls fix that?

Best regards,
Dennis Kreutz

## To Reproduce

Detailed steps to reproduce the behavior:

## Test script or set of commands reproducing this issue

Please provide a test script to reproduce the issue you are reporting, if the 
setup is more complex, GitHub repo links with are also OK.

Issues without a reproduction script are likely to stall and eventually be closed.

## Environment

OS: 
Browser: 
Selenium Grid version 3.17.0v

**Comment by raju249** [comment]
@diemol we can close this issue as this is fixed in #1038 👍 

cc - @AutomatedTester

**Comment by diemol** [comment]
Thanks for point it out @raju249! Closing
